miracle/cosmic-ext-miracle.desktop

```diff
@@ -1,6 +1,6 @@
 [Desktop Entry]
 Name=COSMIC on miracle
 Comment=This session logs you into the COSMIC desktop on miracle
-Exec=/usr/bin/start-cosmic-ext-miracle
+Exec=/usr/local/bin/start-cosmic-ext-miracle
```

Comment:
I don't think this should be the default for the desktop files as most packagers will install to `/usr/bin`. If we wanted to fix this for installing with `just` these files should become templates imo.